python/lsst/meas/algorithms/subtractBackground.py

```diff
@@ -426,5 +426,9 @@ def filterSuperPixels(bbox, background, superPixelFilterSize=3):
         Size of the median filter to use, in pixels.
     """
     statsImg = background.getStatsImage()
+    # scipy's median_filter can't handle NaN values
+    bad = numpy.isnan(statsImg.image.array)
+    if numpy.count_nonzero(bad) > 0:
+        statsImg.image.array[bad] = numpy.nanmedian(statsImg.image.array)
```

Comment:
This will fail if all pixels are NaN - is that ever likely to be a concern? Perhaps, with overly aggressive masking? Is it worth catching that here too?